src/main/scala/notifications/Errors.scala

```diff
@@ -13,8 +13,6 @@ object Errors {
 
   case class AuthorizationError(message: String) extends Throwable with NotificationError
 
-  case class ParseError(message: String) extends Throwable with NotificationError
-
   case class TargetError(message: String) extends Throwable with NotificationError
```

Comment:
I think a better name is required, what does "TargetError" mean? Whats wrong with the target? It didn't parse? Didn't receive message? Or anything else happened to it? Perhaps it could have "receiver" in name instead of "target"

Reply:
My original idea was an error with reaching the email address/slack id, but i couldn't figure out a good name. I changed this to SendError basing on SendFailedException from java mail, hopefully it's clearer now?